internal/scanner/scanner.go

```diff
@@ -100,19 +109,29 @@ func (s *Scanner) ScanNamespace(namespace string) error {
 // auditResource sends the requests to the Policy Server to evaluate the auditable resources.
 // It will iterate over the policies which should evaluate the resource, get the URL to the service of the policy
 // server running the policy, creates the AdmissionReview payload and send the request to the policy server for evaluation
-func auditResource(resource *resources.AuditableResources, resourcesFetcher *ResourcesFetcher, httpClient *http.Client) {
+func auditResource(namespace string, resource *resources.AuditableResources, resourcesFetcher *ResourcesFetcher, httpClient *http.Client) {
 	for _, policy := range resource.Policies {
+		log.Debug().Dict("policy", zerolog.Dict().
+			Str("name", policy.GetName()).
+			Str("UID", string(policy.GetUID()))).
+			Msg("policy")
+
 		url, err := (*resourcesFetcher).GetPolicyServerURLRunningPolicy(context.Background(), policy)
 		if err != nil {
 			// TODO what's the better thing to do here?
 			log.Error().Err(err)
 			continue
 		}
+
 		for _, resource := range resource.Resources {
+			log.Debug().Dict("resource", zerolog.Dict().
+				Str("name", resource.GetName()).
+				Str("UID", string(resource.GetUID()))).
+				Msg("resource under review")
 			admissionRequest := resources.GenerateAdmissionReview(resource)
-			auditResponse, err := sendAdmissionReviewToPolicyServer(url, admissionRequest, httpClient)
-			if err != nil {
-				// TODO what's the better thing to do here?
+			auditResponse, responseErr := sendAdmissionReviewToPolicyServer(url, admissionRequest, httpClient)
+			if responseErr != nil {
+				// log error, will end in PolicyReportResult too
 				log.Error().Err(err)
 				continue
```

Comment:
The `continue` statement has to be removed, otherwise the `report.AddResult` function will not record the error.

Something like that:

```golang
if responseErr != nil {
  log.Error()
} else {
  // need to be inside of else, otherwise the code will panic when trying to access auditResponse.<something>
  log.Debug
}

report.AddResult
```

```

Reply:
Woops, removed `continue` message.